rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -16,6 +16,8 @@ toc::[]
 [id="rosa-q3-2023_{context}"]
 === Q3 2023
 
+* **ROSA CLI update:** The ROSA CLI (`rosa`) was updated to a new version. For information about what's changed in this release, see the link:https://github.com/openshift/rosa/releases/tag/v1.2.25[release notes]. For more information about the ROSA CLI (`rosa`), see xref:../cli_reference/rosa_cli/rosa-get-started-cli.adoc[Getting started with the ROSA CLI].
```

Comment:
LGTM! I was just thinking you could use the shortened reference to rosa in its second mention in the paragraph.

```suggestion
* **ROSA CLI update:** The ROSA CLI (`rosa`) was updated to a new version. For information about what's changed in this release, see the link:https://github.com/openshift/rosa/releases/tag/v1.2.25[release notes]. For more information about the `rosa` command, see xref:../cli_reference/rosa_cli/rosa-get-started-cli.adoc[Getting started with the ROSA CLI].
```

Reply:
Just so you know, I'm new to the peer review squad. My mentor, Shikha Jhala will make one more review pass to ensure I did everything correctly. Thanks!

@sjhala-ccs